mithril-client/src/aggregator_client.rs

```diff
@@ -520,6 +520,26 @@ mod tests {
         )])
     }
 
+    fn mock_server_verifies_accept_encoding_header_values_unsorted(server: &MockServer) {
```

Comment:
_Same kind of comment as in `mithril-signer/src/services/aggregator_client.rs`:_ 

Tests should handle the `status` and `body` themselves and this method could have a simpler name such as `check_accept_encoding_headers`.